docs/source/getting-started/examples/accuracy.ipynb

```diff
@@ -0,0 +1,269 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Accuracy Method"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Many prior examples of queries used comparisons of the actual values to the computed differentially private values. Intuitively, we assessed the quality of our results by visually inspecting the proximity of the actual values to the differentially private outputs. \n",
+    "\n",
+    "However, this approach isn't concrete since interpretations can vary widely on scale, type of visualization, and other factors. A more objective way to evaluate differentially private outputs is by using the `accuracy` method. \n",
+    "\n",
+    "In this notebook ,we will: \n",
```

Comment:
```suggestion
    "In this notebook we will: \n",
```

Reply:
When writing my book, before asking people to review, I used the VSCode Grammarly extension. I think if you used it before requesting reviews, it would help cut down on the noise, so that we can give deeper feedback.